src/util/uploads.js

```diff
@@ -4,6 +4,88 @@
  * @author Box
  */
 
+/**
+ * Returns true if file contains API options
+ *
+ * @param {File | UploadFileWithAPIOptions} item
+ * @returns {boolean}
+ */
+function doesFileContainAPIOptions(file: File | UploadFileWithAPIOptions): boolean {
+    // $FlowFixMe
+    return file.options && file.file;
+}
+
+/**
+ * Returns true if item contains API options
+ *
+ * @param {DataTransferItem | UploadDataTransferItemWithAPIOptions} item
+ * @returns {boolean}
+ */
+function doesDataTransferItemContainAPIOptions(item: DataTransferItem | UploadDataTransferItemWithAPIOptions): boolean {
+    // $FlowFixMe
+    return item.options && item.item;
+}
+
+/**
+ * Converts File or UploadFileWithAPIOptions to File
+ *
+ * @param {File | UploadFileWithAPIOptions} file
+ * @returns {File}
+ */
+function getFile(file: File | UploadFileWithAPIOptions): File {
+    if (doesFileContainAPIOptions(file)) {
+        // $FlowFixMe
+        return file.file;
+    }
+    // $FlowFixMe
```

Comment:
Instead of all these $flowfixme, you should try casting to see if it works.

((file:any):File)

Reply:
👍 will do